web-components/section/section.css

```diff
@@ -0,0 +1,18 @@
+/* * {
+    box-sizing: border-box;
+  }
+   */
+  .row {
+    display: flex;
+  }
+  
+  .column {
```

Comment:
There is a few things wrong with this class:

1. Name - let's change it to 'col', and if it's 50%, it should be col-6.
2. Check what is better to use `width: 50%;` or `flex: 50%;`, and why.
3. `padding` should use value from css variables (`spacer`).
4. `height` - should be 100% of the container. Then, the section itself should have `height` set, we can start with `65vh`.